### ISSUE TITLE
Delete Failed Tracker permanently, check no trackers feature

### ISSUE THREAD

**Issue by JBitte** [BiglySoftware/BiglyBT#2479]
I've been treating failed trackers manually by going into the torrent Library and editing in the Sources Tab of the torrent.
In the My Torrents / All Trackers sidebar, it seems that the only action you can perform is to add a stubborn tracker to a template.

It would be of great benefit if one could hit a nice big red button and delete the offline or failed trackers from the entire instance of biglybt.
This may in fact leave some torrents without an associated tracker, and it would be nice to identify living torrents with no trackers, so that new trackers can be applied

* BiglyBT Version Number B3.0.0.1_B02/4 az3

**Comment by parg** [comment]
The feature already exists - see https://github.com/BiglySoftware/BiglyBT/wiki/All-Trackers-View

**Comment by parg** [comment]
Although the identification of torrents with no trackers is missing - I'll have a think on that

**Comment by parg** [comment]
Beta version 3001_B03 has a new Tag Constraint function that can be used to select downloads that have no trackers:

`countTrackers() == 0`

**Comment by parg** [comment]
Note that you can reapply the tracker removals assigned to your "Delete Now" action by right-clicking on the "Delete Now" Tag and selecting "Tracker Templates... -> reapply"

**Comment by JBitte** [comment]
> Options->Tracker->Client: Enable handling of DNS tracker records used to ameliorate accidental DDOS

I did have this setting enabled.
last month as an all out effort to remove all *.RU domains, I exited gracefully and used BENcode Editor on all the config files of biglybt including alltrackers.config to remove the dead or clogged trackers. Once convinced all references were gone, I restarted biglybt and the dead trackers re-appeared. I thought it might have to do with the references in the ACTIVE Folder, but I'm not an expert
